src/lib/y2network/dialogs/popup.rb

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm/dialog"
+
+module Y2Network
+  module Dialogs
+    # Adapt CWM dialog to allow popup dialogs
+    class Popup < CWM::Dialog
```

Comment:
why copy of Popup class is needed? It is not in SLE15 SP2? If so,  then I think it will be nicer to just backport CWM::Popup to yast2 as expected.

Reply:
You are right, I forgot it was already available and did not checked it.